scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -51,6 +51,12 @@ def call(final pipelineContext) {
     [
       stageName: 'Java 8 Smoke', target: 'test-junit-smoke-jenkins', javaVersion: 8, timeoutValue: 20,
       component: pipelineContext.getBuildConfig().COMPONENT_JAVA
+    ],
+    [
+      stageName: 'Kubernetes', target: 'test-h2o-k8s', timeoutValue: 20,
```

Comment:
this is to be removed before merge correct?

Reply:
Yes.